tests/python/driver/tvmc/test_frontends.py

```diff
@@ -115,26 +115,38 @@ def test_load_model__tflite(tflite_mobilenet_v1_1_quant):
     assert "_param_1" in params.keys()
 
 
-def test_load_model__keras(keras_resnet50):
+def verify_load_model__keras(model, **kwargs):
     # some CI environments wont offer TensorFlow/Keras, so skip in case it is not present
     pytest.importorskip("tensorflow")
 
-    mod, params = tvmc.frontends.load_model(keras_resnet50)
+    mod, params = tvmc.frontends.load_model(model)
     assert type(mod) is IRModule
     assert type(params) is dict
     ## check whether one known value is part of the params dict
     assert "_param_1" in params.keys()
 
 
-def test_load_model__onnx(onnx_resnet50):
-    # some CI environments wont offer onnx, so skip in case it is not present
-    pytest.importorskip("onnx")
+def test_load_model__keras(keras_resnet50):
+    verify_load_model__keras(keras_resnet50)
+    verify_load_model__keras(keras_resnet50, layout="NCHW")
+
 
-    mod, params = tvmc.frontends.load_model(onnx_resnet50)
+def verify_load_model__onnx(model, **kwargs):
```

Comment:
As in the comment above, I would probably suggest using `@pytest.mark.parametrize` here as well and not having the separate `verify_load_model__onnx` helper function.

Reply:
The onnx test is a little bit more complex because of the things being tested, so I think I'm going to leave it as is and try to not break things.